docs/the-git-based-process-you-need-for-mainframe-development-DRAFT/introduction.md

```diff
@@ -32,6 +32,11 @@ The target audience of this document are Mainframe DevOps architects and SCM spe
 
 Some aims and assumptions which guided our recommendations include:
 
+- We are focusing on recommending an effective branching strategy and workflow. Beyond relying on `git` and the facilities commonly
+provided by hosted `git` service providers, we make few assumptions about the tools chosen  - except, obviously IBM Dependency-Based Build 
+and IBM UrbanCode Deploy. 
```

Comment:
We also need to list **IBM Wazi Deploy** as a deployment technology.

Reply:
I considered that but unsure if we had anything concrete to say about Wazi Deploy - but I've added it plus included some links for each one we list.